src/devices/honeywell.c

```diff
@@ -46,20 +64,28 @@ static int honeywell_callback(bitbuffer_t *bitbuffer) {
 
     channel = bb[2] >> 4;
     device_id = ((bb[2] & 0xf) << 16) | (bb[3] << 8)| bb[4];
+
     event = bb[5];
-    state = (event & 0x80) >> 7;
-    heartbeat = (event & 0x04) >> 2;
+
+    if(event == 0x20) add_motion_detector(device_id);
```

Comment:
It seems as though the logic from the [source](https://github.com/jhaines0/HoneywellSecurity/blob/18a4ac82e79cca532b1eadb33327d49a73ce7df1/rpi/digitalDecoder.cpp#L73) is slightly different, `event > 0x20`, is this intentional?

Reply:
the logic from the source is different; but I have both motion sensors and door/window sensors at my house, and this is the code that appears to work for me.  OTOH, if you read through the issues in the referenced project (that cover the sensor encoding), the code here is definitely not successfully handling all potential cases: https://github.com/jhaines0/HoneywellSecurity/issues/2 .  However, since the previous honeywell.c code didn't appear to handle motion sensors at all; and since it works for me with both, I opened this PR.  I was actually gonna see if I could get the original honeywell.c contributor (adam1010) to try out my branch; but he seem's to've dropped off of github (or at least, prevented himself from being @-ed in commentary).  I can keep the honeywell changes local to my forked copy of rtl_433 if you like; but I'd still really like to get the json-formatting changes committed: so I can open a different PR for that if necessary.

Reply:
> the code here is definitely not successfully handling all potential cases: jhaines0/HoneywellSecurity#2

Yep, as it turns out I've been using this `securityguy`'s fork of HoneywellSecurity in my own house (and it forked it [here](https://github.com/mcolyer/Honeywell5800)) as it seems to work with more of my sensors. 

My motion sensor seems to work well with rtl_433's Honeywell protocol but it doesn't seem to see any messages from my 5816 door sensors. Even if I disable the checksum messages it doesn't seem to correctly decode the 5816 byte stream. When I turn on debug mode in rtl_433 it says something about thinking it's a frequency shift encoding but I don't think that's right. I could try to debug further but I'd need some pointers about what to look for next.

> I can keep the honeywell changes local to my forked copy of rtl_433 if you like; but I'd still really like to get the json-formatting changes committed: so I can open a different PR for that if necessary.

No need to keep them out, as this change doesn't negatively impact my house as it didn't recognize my 5816's before this change either - just wanted to make sure this was intentional.

Reply:
@mcolyer do you want to collaborate on further fixing up honeywell.c so that it works for both of us/incorporates more functionality referred to in jhaines0's project?